release/pkg/generate_spec.go

```diff
@@ -267,6 +273,7 @@ func (r *ReleaseConfig) GenerateBundleArtifactsTable() (map[string][]Artifact, e
 		"kube-rbac-proxy":                 r.GetKubeRbacProxyAssets,
 		"kube-vip":                        r.GetKubeVipAssets,
 		"flux":                            r.GetFluxAssets,
+		"eks-anywhere-packages":           r.GetPackagesAssets,
```

Comment:
Same comment for this, refer to line 287 below.

Reply:
Fixed in ee9b00a